### ISSUE TITLE
Bug: don’t sort `() => …` type in `perfectionist/sort-interfaces`

### ISSUE THREAD

**Issue by ai** [azat-io/eslint-plugin-perfectionist#30]
### Describe the bug

`perfectionist/sort-interfaces` is useful to sort keys.

But interface can also contain function arguments like in code example below.

Changing the order of these calls could break API very easy (order is very important there).

I suggest counting them all as `()` and don’t sort between each other (sort only keys in interface).

### Code example

```ts
interface Count {
  <Parameters extends Record<string, number | string>>(
    input: TranslationFunction<[Parameters], string>
  ): TranslationFunctionAlternatives<Parameters>
  <Input extends CountInput>(input: Input): TranslationFunction<
    [number],
    Input[keyof Input]
  >
}
```

### ESLint version

v8.44

### ESLint Plugin Perfectionist version

v0.9.0

### Validations

- [X] Read the [docs](https://eslint-plugin-perfectionist.azat.io).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Yep, we should not sort interface members that are TypeScript call signature declarations, thanks for report.

I will fix it asap.

**Comment by azat-io** [comment]
Fixed in 5829a654f2f88c9cb42ba847d6ee5b8d1a0391d5

Released in [v1.5.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v1.5.0)

**Comment by ai** [comment]
Thanks. I migrated my project to new version and everything is goog right now.
